docs/docs/middlewares/path.md

```diff
@@ -0,0 +1,64 @@
+# Path
+This function returns the URL for the given route. 
+
+## Parameters
+```path(page, queries, locale)```
+
+parameter        | type           | required | description
+-----------------|----------------|----------|------------
+page             | string         | ✓        | Key of current page listen in `site.yml` (e.g. `index` or `contact`).
+queries          | array          | ❌        | Map of url parameters (e.g. `{ 'product_slug': product.slug }`)
+locale           | string         | ❌        | Language-Territory code (e.g. `cz_CZ`).
+
+**Example**
+
+```
+# file.html.twig
+
+…
+<a href="{{ path('index') }}">
+    Back to homepage
+</a>
+…
+
+```
+
+**Example**
+
+This example shows how you can set path to detail page contents information about specific product.
+The specific product is here clearly identified by `slug` which must be key in relevant `.yml` file
+(in this case `products.yml`, which has to be listed in `site.yml`).
```

Comment:
I'd prefer it like this:

_In this example you can see how to set path to page with product details. The product is identified by `slug` which is validated against corresponding YAML file containing all products (here `products.yml` assigned to `data.products` variable in `site.yml`)._